pkg/block/block.go

```diff
@@ -29,6 +29,7 @@ const (
 	DRIVE_TYPE_FDD               // Floppy disk drive
 	DRIVE_TYPE_ODD               // Optical disk drive
 	DRIVE_TYPE_SSD               // Solid-state drive
+	DRIVE_TYPE_LOOP              // loop device
```

Comment:
I'm questioning whether "loop" should be a DRIVE_TYPE... is it really a type of drive? I'm not sure... maybe we should make a new `DRIVE_TYPE_VIRTUAL` or `DRIVE_TYPE_PSEUDO`.

Reply:
OK, let's go with DRIVE_TYPE_VIRTUAL then, @Itxaka.